### PR TITLE
#3-RT-Blurbcard-Compoent

### PR DESCRIPTION
**Pull request by DJMF2015**
Added blurb card component. Main need some changes. eg) had to split the three paragraphs into different props as very long...

Screenshot 2022-10-13 at 12 48 07

### CODE REVIEW

**Review by josephshambrook — CHANGES_REQUESTED** [review]

File: pages/index.js [hunk 1]
```diff
@@ -1,17 +1,47 @@
...
-    <Layout>
-      <Head>
-        <title>Rock Trust</title>
-        <meta name="description" content="Generated by create next app" />
-        <link rel="icon" href="/favicon.ico" />
-      </Head>
-
-      <h1>Rock Trust!</h1>
-    </Layout>
+    <>
```
Comment (josephshambrook) [inline pages/index.js:7]: Don't need a fragment here as only one top-level element (`<Layout>`) is being returned

File: pages/index.js [hunk 2]
```diff
@@ -1,17 +1,47 @@
...
+        </Head>
+        <h1>Rock Trust!</h1>
+        <BlurbCard
+          imgLogo={
+            "https://rt-dev.tfhd.net/wp-content/uploads/2022/08/rocktrustlogofull.png"
+          }
+          donateBtn={"Donate Now"}
+          findOutMoreBtn={"Find Out More"}
+        >
+          {" "}
```
Comment (josephshambrook) [inline pages/index.js:22]: This can be deleted

File: pages/index.js [hunk 3]
```diff
@@ -1,17 +1,47 @@
...
+          </p>
+          <p className="m-0 p-1.5 pb-6">
+            We advise, educate and support young people to enable them to build
+            the personal skills and resources required to make a positive and
+            healthy transition to adulthood. We also work to ensure that the
+            public, policy makers, commissioners and practitioners understand
+            the issues, make decisions and take action which will end youth
+            homelessness"
+          </p>
+          <p className="m-0 p-1.5 text-lg pb-6">
```
Comment (josephshambrook) [inline pages/index.js:37]: This one has a class of `text-lg` but the rest don't. You would probably want to add this class to the others as well, or remove it from this one.

**Review by josephshambrook — COMMENTED** [review]
I'll merge this with main to get rid of the conflicts

File: pages/index.js (hunk 3 above)
Comment (DJMF2015) [reply pages/index.js]: Iam sorry joe i never saw this comment and have got into merge conflicts

File: pages/index.js (hunk 1 above)
Comment (DJMF2015) [reply pages/index.js]: i  have removed this

File: pages/index.js (hunk 3 above)
Comment (DJMF2015) [reply pages/index.js]: I have removed the text-lg as i don't feel it is needed looking at the current designs.

File: pages/index.js (hunk 2 above)
Comment (DJMF2015) [reply pages/index.js]: Have left alone as wasn't sure why i would want to delete the buttons.

File: components/Blurb.js [hunk 4]
```diff
@@ -0,0 +1,28 @@
...
+  findOutMoreBtn,
+}) {
+  return (
+    <div className="container m-10 max-w-3xl">
+      <>
+        <div className="flex-wrap">
+          <img className="m-1 py-10" src={imgLogo} alt="rocktrust logo full" />
+          <h3 class="m-0 p-1.5 text-lg pb-6">{blurbPara1}</h3>
+          <h3 class="m-0 p-1.5 text-lg pb-6">{blurbPara2}</h3>
+          <h3 class="m-0 p-1.5 text-lg pb-6">{blurbPara3}</h3>
```
[the remark this answers is not in the document]
Comment (DJMF2015) [reply components/Blurb.js]: This bit has been done as per comments.

File: pages/index.js (hunk 2 above)
Comment (josephshambrook) [reply pages/index.js]: I didn't mean the buttons, I meant the `{" "}` line

**Review by josephshambrook — APPROVED** [review]

**Pull request merged**